config/e2e/global_operator.yaml

```diff
@@ -157,6 +157,8 @@ spec:
   serviceName: {{ .GlobalOperator.Name }}
   template:
     metadata:
+      annotations:
+        "co.elastic.logs/raw": "[{\"type\":\"container\",\"json.keys_under_root\":true,\"paths\":[\"/var/log/containers/*${data.kubernetes.container.id}.log\"],\"processors\":[{\"convert\":{\"mode\":\"rename\",\"ignore_missing\":true,\"fields\":[{\"from\":\"error\",\"to\":\"_error\"}]}},{\"convert\":{\"mode\":\"rename\",\"ignore_missing\":true,\"fields\":[{\"from\":\"_error\",\"to\":\"error.message\"}]}},{\"convert\":{\"mode\":\"rename\",\"ignore_missing\":true,\"fields\":[{\"from\":\"source\",\"to\":\"_source\"}]}},{\"convert\":{\"mode\":\"rename\",\"ignore_missing\":true,\"fields\":[{\"from\":\"_source\",\"to\":\"event.source\"}]}}]}]"
```

Comment:
Is there a way we could have this in a single place?

Reply:
I was wondering the same. I'm not sure it is easy since the manifests are handled by different templating systems.

Reply:
This can be delayed to when we decide to handle https://github.com/elastic/cloud-on-k8s/issues/2254 and/or https://github.com/elastic/cloud-on-k8s/issues/2406.

Reply:
Can you comment (on all 5 files) what this is doing?